data/wanderer/wanderers start.txt

```diff
@@ -224,10 +224,12 @@ mission "Wanderers: Diplomacy"
 
 
 mission "Wanderers: Translation Machine"
-	description "Sayari, the Hai ambassador, suggested that you visit <destination>, where Hai technologists are working on translation machines."
+	description "Find language training data at <destination> so the Wanderers can create an automated language translator."
 	landing
 	source "Vara K'chrai"
-	destination "Greenwater"
+	destination "Alexandria"
+	substitutions
+		"<Library search>" `You travel through the library searching for data suitable for training the Wanderer translation algorithm, but you have an unexpected problem: you're surrounded by it. Even fifty Bulk Freighters would be unable to carry all the tomes and data cores of Alexandria library. Someone with expertise in natural language processing would know what to do. Asking around, you're referred to a research librarian who develops automated translation algorithms for dead languages.`
```

Comment:
I think, in your changes, "guessed" should be "guess" because the rest of the experience is in present tense. Recall that Captain Last may not have ever been to Alexandria Library, and thus may have no idea how many Bulk Freighters would be needed to transport it until arriving at the library.

Reply:
Also, I'm not a fan of telling the player what they think, since it restricts the paths their mind can take when perceiving the story. Instead of saying what they guess, I'd rather describe the surroundings.

Reply:
Definitely don't want to tell the player what to think, although the former makes it seem like <first> <last> knows for a certainty what 50 Bulk Freighters could carry, which feels more restrictive. The "You guess" adds ambiguity to an otherwise specific number. I'd either go with that, or just a general "fleet of" Bulk Freighters personally.

Reply:
Well, fifty Bulk Freighters is something like 12,500 tons. That's information you can get in the game. I thought 50 was a big enough number that was ambiguous enough that it sounded outlandish, but I suppose "a fleet", or even bigger, "an armada", is good enough?

Reply:
Yee, it's often a problem of mine with writing too long, using past tenses.

As for the thought, I'm not too much a fan of being told information, so it's a minor compromise for the Reader to know what to think.

Reply:
I changed "guessed" to "guess," but I'm not hearing consensus on any other wording.